### DISCUSSION TITLE
Homebrew need parallel install support at least for downloads

### DISCUSSION THREAD

**Discussion by ssbarnea** [Homebrew/discussions#3826] [Everyday usage]
As there were multiple issues that were closed, maybe it is time to create discussion thread for parallel downloading and eventually even installing of casks.

Due to its serial behavior, brew is one of the slowest package managers and we should try to address this in the future, even if that is not very easy. At least the download part should be ok to happen in background.

* https://github.com/Homebrew/brew/issues/1865
* https://github.com/Homebrew/brew/issues/11363
* https://github.com/Homebrew/brew/issues/12489
* https://github.com/Homebrew/brew/issues/13064
* https://github.com/Homebrew/homebrew-cask/issues/21992

**Comment by SMillerDev** [comment]
Go ahead

**Comment by carlocab** [comment]
This is being tracked at https://github.com/Homebrew/brew/issues/18278.
